ansible/roles/analytics-spark-provision/templates/jets3t.j2

```diff
@@ -1,3 +1,16 @@
-storage-service.request-signature-version=AWS4-HMAC-SHA256
-s3service.s3-endpoint=s3-ap-south-1.amazonaws.com
+storage-service.request-signature-version={{ jets3t_s3_request_signature_version }}
+s3service.s3-endpoint={{ jets3t_s3_endpoint_host }}
+
+{% if jets3t_s3_disable_dns_buckets %}
+s3service.disable-dns-buckets=true
```

Comment:
spark s3 connector does not hit the custom endpoint until the value for `s3service.s3-endpoint` is configured in `jets3t.j2`. same goes for the other configuration. 

`jets3t_s3_disable_dns_buckets` can be directly rendered instead of the if statements (motivation was, that the `s3service.disable-dns-buckets` config is not needed in case of AWS S3), will update this accordingly

Reply:
removed jinja if directive